src/chapter3/linux-cmds.md

```diff
@@ -1 +1,45 @@
 # Linux Commands
+
+Even if you are already familiar with linux, please read through all of these commands, as some are specific to M3.
+
+## Basic Linux Commands
+
+| Command | Function |
+| --- | --- |
+| `pwd` | prints current directory |
+| `ls` | prints list of files / directories in current directory (add a `-a` to list everything, including hidden files/directories |
+| `mkdir` | makes a directory |
+| `rm <filename>` | deletes *filename*. add `-r` to delete directory. add `-f` to force deletion (be really careful with that one) |
+| `cd <directory>` | move directory. put `..` to go up one directory (to the parent of your current one) |
```

Comment:
Move the stuff about `..` and `.` into a note below the table like: 'When navigating directories `..` and `.` are used to denote the parent and current directory'. I say this for two reasons one, `..` and `'` can be used in other commands like `rm`, not just `cd` because they are a more general Bash construct not just a CLI thing. Second, this gives opportunity to talk about `/`, `~` briefly and even relative directories.